mate-extra/mate-polkit/mate-polkit-1.27.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
This isn't something I would've expected you to spot yet, as it's a bit niche, but:
```
+if  test "x$enable_appindicator" == "xauto" &&
+   (test "x$have_appindicator_ayatana" == "xyes" ||
+    test "x$have_appindicator_ubuntu" == "xyes"); then
```

These should be `=`, not `==`, or it's going to break with dash as /bin/sh. We should send that upstream.